index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="name">Furkan Salim Hıdır</div>
+        <img id="burger-menu" src="union.png" alt="burger-menu">   
+    </header>
+    <section>
+        <h1 >Hey There.<br>I’m Salim</h1>
+        <h2>I’m a Software Developer</h2>
+        <p>I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need to be coded, 
+            don’t hesitate to contact me.<br>
+        <img id="social-buttons" src="SocialMedia.png" alt="social"> </p>
```

Comment:
- [ ] I would recommend using icon instead of images and separate icon for different social icon. Also, the list of icons should use `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). It will also helps screen readers and other technologies that depend on correct semantics to work. You can search for icon from [here as well](https://fontawesome.com/icons). :smile:

Reply:
I liked the website so much and tried to use them. Thank you😄